pkg/nvml/nvml.go

```diff
@@ -25,2283 +25,2522 @@ package nvml
 #include "cgo_helpers.h"
 */
 import "C"
-import "unsafe"
 
 // nvmlInit_v2 function as declared in nvml/nvml.h:1766
-func nvmlInit_v2() Return {
-	__ret := C.nvmlInit_v2()
-	__v := (Return)(__ret)
-	return __v
+func nvmlInit_v2() Return{
+__ret := C.nvmlInit_v2()
+__v := (Return)(__ret)
+return __v
```

Comment:
What are all these changes in this file from?

Reply:
Was also due to the insufficient disk space. I missed the warnings / errors.